varats-core/varats/experiment/wllvm.py

```diff
@@ -48,6 +48,7 @@ class BCFileExtensions(Enum):
     TBAA = "TBAA"
     FEATURE = 'feature'
     BLAME = "blame"
+    ARCH = "fvara-arch"
```

Comment:
I think `architecture` is a better fitting term here.
```suggestion
    ARCH = "architecture"
```